tests/Fixtures/app/AppKernel.php

```diff
@@ -47,6 +47,13 @@ public function registerBundles()
 
     public function registerContainerConfiguration(LoaderInterface $loader)
     {
-        $loader->load($this->getRootDir().'/config/config.yml');
+        $environment = $this->getEnvironment();
+
+        // patch for behat not supporting %env(APP_ENV)% in older versions
+        if ($appEnv = $_SERVER['APP_ENV'] ?? null && $appEnv !== $environment) {
```

Comment:
Should not it be `if (($appEnv = $_SERVER['APP_ENV'] ?? null) && $appEnv !== $environment) {`?

Reply:
Sorry, fixed by #1524.